.idea/runConfigurations/Generate_FIR_Checker_Boilerplate.xml

```diff
@@ -0,0 +1,24 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Generate FIR Checker Boilerplate" type="GradleRunConfiguration" factoryName="Gradle">
```

Comment:
It also generates diagnostics for IDEA, so I would name it smth like `Generate FIR Checker Components and FIR/IDE Diagnostics`